meeting-notes/2018-01-29-js-ipfs-dev-team-weekly.md

```diff
@@ -0,0 +1,94 @@
+# ⒿⓈ⚡️  js-ipfs Core Dev Team Weekly 2018-01-29
+
+- **Lead:** David Dias (@diasdavid)
+- **Notetaker:** @vmx
+- **Attendees:**
+  - @dryajov
+  - @lidel
+  - @powersource
+  - @Jaredhardy
+  - @richard schneider
+  - @jonnycrunch
+  - @rob
+  - @vmx
+  - @victor
+  - @hacdias
+
+## Agenda
+
+- Re-enter Weekly Sync Mode (@diasdavid)
+  - What we expect to achieve with these calls
+  - How these calls are going to function
+- Pair Programming Sessions (@diasdavid)
+- Round of updates (@all)
+  - Done: What have you accomplished since the last Weekly?
+  - Blocked: Were there any blockers? If so, which ones? Is it still blocked? Why?
+  - Next: What is the next important thing you should focus on?
+- Plan this week (@all)
+  - Select issues to work on
+- Review remaining issues (by priority) if there is time left (@all)
+
+## Notes
+
```

Comment:
@vmx could you add a link to the recording https://www.youtube.com/watch?v=F0wEg6ZQB34?